### PR TITLE
chore(docs): update operational commands, cdn, static site pages

### PR DESCRIPTION
**Pull request by huanjani**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/commands/svc-logs.en.md [hunk 1]
```diff
@@ -5,20 +5,25 @@ $ copilot svc logs
...
-`copilot svc logs` displays the logs of a deployed service.
+`copilot svc logs` displays the logs of a deployed service.  
+(Logs are not available for Static Site services.)
 
 ## What are the flags?
 
 ```
-  -a, --app string          Name of the application.
+  -a, --app string          Name of the application. (default "testing-buildspec")
+      --container string    Optional. Return only logs from a specific container.
```
Comment (iamhopaul123) [inline site/content/docs/commands/svc-logs.en.md:15]: Seems like it's not updated for quite a long time 🤦

File: site/content/docs/concepts/services.en.md [hunk 2]
```diff
@@ -152,27 +151,47 @@ About
...
+  COPILOT_SERVICE_DISCOVERY_ENDPOINT    "        test         test.my-app.local
+  COPILOT_SERVICE_NAME                  "        test         front-end
+  
+Secrets
+
+  Name                   Container  Environment  Value
+  ----                   ---------  -----------  -----
+  GITHUB_WEBHOOK_SECRET  front-end  test         parameter/GH_WEBHOOK_SECRET
 ```
+The output of `copilot svc show` varies depending on your service type. For example, the summary for a __Static Site__ includes a tree representation of your S3 bucket's contents.
```
Comment (iamhopaul123) [inline site/content/docs/concepts/services.en.md:194]: Maybe an example for the tree below?

**Review by iamhopaul123 — APPROVED** [review]
💯

**Review by KollaAdithya — APPROVED** [review]
lgtm!

**Pull request merged**